src/services/postService.js

```diff
@@ -1,12 +1,41 @@
 const { throwIfError } = require('../utilities/dynamoUtilities');
-const { sendPost } = require("../repository/postDAO");
+const postDAO = require("../repository/postDAO");
+const uuid = require("uuid");
 
 async function createPost(username, text, score, title){
-    const post = await sendPost(username, text, score, title);
-    throwIfError(post);
-    return {message: "Post created successfully"};
+    const post = {class: "post", itemID: uuid.v4(), by: username, desc: text, score, title, replies: []};
+    const data = await postDAO.sendPost(post);
+    throwIfError(data);
+    return post;
+}
+
+async function seePosts(){
+    //TODO try to see if better option than scan
+    return await postDAO.scanPosts();
```

Comment:
I don't think there's a better option than scan here unless you want to filter out posts by some attribute because we'd want to see all the posts in the database to display to users.